src/bridge/SequencerInbox.sol

```diff
@@ -374,26 +374,32 @@ contract SequencerInbox is DelegateCallAware, GasRefundEnabled, ISequencerInbox
         IGasRefunder gasRefunder,
         uint256 prevMessageCount,
         uint256 newMessageCount,
-        bytes memory quote
+        bytes memory batcherSignatureAndHotshotHeight
     ) external refundsGas(gasRefunder, IReader4844(address(0))) {
         // solhint-disable-next-line avoid-tx-origin
         if (msg.sender != tx.origin) revert NotOrigin();
         if (!isBatchPoster[msg.sender]) revert NotBatchPoster();
 
         // take keccak2256 hash of all the function arguments except the quote
```

Comment:
Nit: we should probably update and move this comment.

Reply:
https://github.com/EspressoSystems/nitro-contracts/pull/53/commits/686f7ae66599fa385320854c9344934676ee970b